src/bin/edit/main.rs

```diff
@@ -250,7 +250,8 @@ fn handle_args(state: &mut State) -> apperr::Result<bool> {
             break;
         }
         let p = cwd.join(Path::new(&arg));
-        let p = path::normalize(&p);
+        // `p` is not absolute when `arg` is a drive letter path like `C:`
+        let p = if p.is_absolute() { path::normalize(&p) } else { p };
```

Comment:
Instead of fixing it here, `path::normalize` should not panic in the first place.